src/main/java/com/epam/paymentsystem/controller/BankAccountController.java

```diff
@@ -0,0 +1,49 @@
+package com.epam.paymentsystem.controller;
+
+import com.epam.paymentsystem.service.BankAccountService;
+import com.epam.paymentsystem.service.model.BankAccount;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+public class BankAccountController {
+    private BankAccountService service;
+
+    @Autowired
+    public BankAccountController(BankAccountService service) {
+        this.service = service;
+    }
+
+    @ResponseStatus(HttpStatus.OK)
+    @GetMapping(value = "/account/{bankAccountID}")
```

Comment:
I would recommend using @RequestMapping("/account") instead of duplicate same url in each method. Please check other controllers, I see the same issue there